packages/utilities/psammead-assets/package.json

```diff
@@ -23,6 +23,13 @@
     "amp"
   ],
   "devDependencies": {
```

Comment:
I think this needs `prop-types` in dev deps

Reply:
Just as a side note. We might need to update `package.json` of other packages that use `prop-types`. They don't seem to include `prop-types` as a `devDependency`